hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/BaseHoodieWriteClient.java

```diff
@@ -568,6 +569,30 @@ protected void postCommit(HoodieTable table, HoodieCommitMetadata metadata, Stri
       // Delete the marker directory for the instant.
       WriteMarkersFactory.get(config.getMarkersType(), table, instantTime)
           .quietDeleteMarkerDir(context, config.getMarkersDeleteParallelism());
+      if (metrics.getClusteringInstantTimerCtx() != null) {
```

Comment:
Check Config `config.isMetricsOn()`  instead of trying to create a Timer.
What we want is to monitor clustering and compaction instant here rat